src/projetos/HomeProjetosListPage.tsx

```
@@ -226,84 +244,76 @@ export function HomeProjetosListPage() {
-                      </PaginationItem>
-                    ))}
+                {/* Lista de cards */}
+                <div className="flex flex-wrap gap-6 justify-center w-full">
+                  {loading
+                    ? Array.from({ length: itensPorPagina }).map((_, index) => (
+                      <div key={`loading-${index}`} style={{ width: '500px' }}>
+                        <LoadingCard />
+                      </div>
+                    ))
+                    : paginatedItems.length > 0
+                      ? paginatedItems.map((projeto: any) => (
```

ESLint

[error] Replace `··:·paginatedItems.length·>·0⏎······················?` with `)·:·paginatedItems.length·>·0·?·(⏎···················`

(prettier/prettier)